sdks/java/core/src/main/java/org/apache/beam/sdk/options/InvokePipelineOptionsFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.beam.sdk.options;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.Iterators;
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+
+@SuppressWarnings("rawtypes")
+/**
+ * Interface to support offspring wire-in for Li
```

Comment:
Can you elaborate more on how this interface is supposed to be used, as the term `InvokePipelineOptionsFactory` seems ambiguous to me.

Reply:
Add some comments.
So this interface is to designed to check if the input class meets the requirements to initialize some other object underneath to support further execution.